toast-tk-web-agent/src/main/java/io/toast/tk/agent/run/TestRunner.java

```diff
@@ -0,0 +1,104 @@
+package io.toast.tk.agent.run;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.util.Strings;
+
+import com.google.inject.Module;
+
+import io.toast.tk.agent.config.AgentConfigProvider;
+import io.toast.tk.agent.ui.NotificationManager;
+import io.toast.tk.dao.domain.impl.test.block.ITestPage;
+//import io.toast.tk.dao.domain.impl.test.block.ITestPlan;
+import io.toast.tk.plugin.IAgentPlugin;
+import io.toast.tk.plugin.PluginLoader;
+import io.toast.tk.runtime.parse.FileHelper;
+import io.toast.tk.runtime.parse.TestParser;
+
+public class TestRunner {
+	private static final Logger LOG = LogManager.getLogger(TestRunner.class);
+	private TestPageRunner testPageRunner;
+	private AgentConfigProvider provider;
+	public String fileName;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make fileName a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)